indico/modules/events/management/templates/clone_event.html

```diff
@@ -91,8 +91,13 @@
                             {% set count %}
                                 <span class="count"></span>
                             {% endset %}
+                            {% set info %}
+                                <span class="info"></span>
+                            {% endset %}
                             {% trans link='<a class="event-list">'|safe, endlink="</a>"|safe -%}
-                                {{ count }} new events will be created. {{ link }}When?{{ endlink}}
+                                {{ count }} new events will be created.
+                                <strong>{{ info }}</strong>
```

Comment:
this feels dirty since we are combining two translated strings. i don't see much better options here

Reply:
actually, if you returned just a flag about using last-day-of-month instead of a string, you could simply have two translated strings here instead of having to combine strings.

Reply:
Do you think it's better to use a flag? I considered that at first, but then I figured that a variable is more generic and could be used for some other messages as well (if ever we would need them :) )

Reply:
The disadvantage of the free-text message is that you need to concatenate two translated messages which may or may not sound weird in other languages.

Reply:
Yeah, that's a fair reason. Thanks